### PR TITLE
Build job worker image

### PR DESCRIPTION
**Pull request by cob16**
- adds a new step to build the new 'worker' image
- prefixes tags with the type of image
- changes deploy scripts so that deploy tag is passed down

*New image is uploaded but not deployed to envs yet*

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -41,24 +42,37 @@ jobs:
...
 
 workflows:
   version: 2
   release:
     jobs:
       - test
-      - aws-ecr/build_and_push_image:
-          account-url: "754256621582.dkr.ecr.eu-west-2.amazonaws.com"
+      - aws-ecr/build-and-push-image:
+          name: push_app_image #todo remove ECR_ENDPOINT var as it is not used
```
Comment (cob16) [inline .circleci/config.yml]: will do this todo once merged into master (just incase it was being used)

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 version: 2.1
 orbs:
-  aws-ecr: circleci/aws-ecr@0.0.2
+  aws-ecr: circleci/aws-ecr@6.3.0
```
Comment (emileswarts) [inline .circleci/config.yml]: Nice version bump

File: .circleci/config.yml [hunk 3]
```diff
@@ -41,24 +42,37 @@ jobs:
             - "7e:e7:44:be:a1:4b:4e:3c:56:65:3a:94:19:50:02:cc"
       - run:
           name: deploy to production
-          command: './deploy/scripts/circle_deploy.sh production $KUBE_TOKEN_PRODUCTION'
+          command: './deploy/scripts/circle_deploy.sh "APP_${CIRCLE_SHA1}" production $KUBE_TOKEN_PRODUCTION'
```
Comment (emileswarts) [inline .circleci/config.yml]: Interesting, why the APP_ prefix?

Comment (emileswarts) [reply .circleci/config.yml]: Nevermind, I see

**Comment by emileswarts** [comment]
Nice, looks good

**Pull request merged**